### PR TITLE
Dynamically modify a relative path to a full path for two failing test cases

### PR DESCRIPTION
**Pull request by Randy-Ma**
### Purpose

This submission is to make it robust to get the full path from a relative path in the filename node. Right now, the relative path will be converted to a full path by using the current directory. It may cause issues when the current directory is not the desired one for the relative path.

The change here is to adjust the path in the test cases after the files are opened. BTW, as in one test case, the file is set to run automatically, the graph runs before the change to the path. So the file is set to run manually.

### Declarations

Check these if you believe they are true

- [X] The code base is in a better state after this PR

### Reviewers

@Benglin 

### FYIs

@monikaprabhu

### CODE REVIEW

File: test/Libraries/RevitIntegrationTests/WorkflowTest.cs [hunk 1]
```diff
@@ -198,12 +206,8 @@ public void Test_PanelWall()
 
             //check AdaptiveComponent.ByPoint
             var acID = "85e957b8-90b7-46a5-823c-18aaa4155c98";
-            AssertPreviewCount(acID, 60);
```
Comment (Randy-Ma) [inline test/Libraries/RevitIntegrationTests/WorkflowTest.cs]: This does not reflect the real situation in the graph. Here is should be a single element.

File: test/System/Workflow/PerforatedScreenByImage/PanelWall.dyn [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<Workspace Version="0.8.1.1310" X="-329.867792183962" Y="-810.6891383816" zoom="0.653999019844256" Name="Home" RunType="Automatic" RunPeriod="100" HasRunWithoutCrash="True">
+<Workspace Version="0.8.1.1310" X="-329.867792183962" Y="-810.6891383816" zoom="0.653999019844256" Name="Home" RunType="Manual" RunPeriod="100" HasRunWithoutCrash="True">
```
Comment (sharadkjaiswal) [inline test/System/Workflow/PerforatedScreenByImage/PanelWall.dyn]: Any reason for the change in RunType? we want to ensure that our tests run in automatic execution mode.

**Comment by Benglin** [comment]
Thanks for fixing this, apart from the point brought up by @sharadkjaiswal, LGTM.

File: test/System/Workflow/PerforatedScreenByImage/PanelWall.dyn (hunk 2 above)
Comment (Randy-Ma) [inline test/System/Workflow/PerforatedScreenByImage/PanelWall.dyn]: If we do not change the RunType, before we have a chance to change the path in the filename node, the graph already runs and fails.

**Comment by monikaprabhu** [comment]
@Benglin, Please hold on do not merge- I think there is a bug introduced in the submission which is why the tests (Test_Panels and Test_PanelWall)are failing! Zora's original verification on PanelWall and Panels. dyn are correct. There is a change in behaviour in List.FirstItem which is probably why the tests are failing.I need more time to test this.

**Comment by Randy-Ma** [comment]
@Benglin 
I have reverted the part impacted by a specific package.

**Pull request closed without merging**